examples/gstreamer_slint_demo/Cargo.toml

```
@@ -0,0 +1,30 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: MIT
+
+[package]
+name = "gstreamer_slint_demo"
+version = "1.4.0"
+authors = ["Slint Developers <info@slint.dev>"]
+edition = "2021"
+build = "build.rs"
+license = "MIT"
+publish = false
+
+[[bin]]
+path = "main.rs"
+name = "gstreamer_slint_demo"
+
+[dependencies]
+slint = { path = "../../api/rs/slint" }
+anyhow = { version = "1.0" }
+futures = { version = "0.3.28" }
+
+gst = {package="gstreamer", version="0.21.3"}
+gst-audio = {package="gstreamer-audio",version="0.21.3"}
+gst-video = {package="gstreamer-video",version="0.21.2"}
+gst-app = {package="gstreamer-app",version="0.21.2"}
```

Just some nitpicking ;)

```suggestion
gst = {package = "gstreamer", version = "0.21.3"}
gst-audio = {package = "gstreamer-audio", version = "0.21.3" }
gst-video = {packag e= "gstreamer-video", version = "0.21.2" }
gst-app = {package = "gstreamer-app", version = "0.21.2" }
```

---

Ooops, thanks  - fixed :)